### PR TITLE
GAEC - Added Hashing option 

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

Added hashing option in the connection settings

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: destinations/advertising/google-adwords-enhanced-conversions.mdx [hunk 1]
```diff
@@ -63,9 +63,9 @@ For more information on subaccounts, refer to the <a href="https://support.googl
 This field is required only when you want to send data to a customer list of a subaccount.
 </div>
 
-### Event settings
```
Comment (ItsSudip) [inline destinations/advertising/google-adwords-enhanced-conversions.mdx:66]: We don't need to remove this.

File: destinations/advertising/google-adwords-enhanced-conversions.mdx [hunk 2]
```diff
@@ -63,9 +63,9 @@ For more information on subaccounts, refer to the <a href="https://support.googl
...
+- **List of Conversion**: Enter the list of events on which you want to perform enhanced conversions. RudderStack will discard any events that are not specified in this list.
 
-**List of Conversion**: Enter the list of events on which you want to perform enhanced conversions. RudderStack will discard any events that are not specified in this list.
+- **Hashing Required**: Enable this setting to hash encrypt the user identifiers, such as email, phone number, first name, last name and street address.
```
Comment (ItsSudip) [inline destinations/advertising/google-adwords-enhanced-conversions.mdx:68]: Add another line. `By default, we are enabling this.`

File: destinations/advertising/google-adwords-enhanced-conversions.mdx (hunk 1 above)
Comment (ishachopra0401) [reply destinations/advertising/google-adwords-enhanced-conversions.mdx:66]: There is only one setting under 'Event Settings'. Also, Hashing required option does not come under 'Event Settings', it's separate. We might need to add another heading for 'Hashing required' then. Hence, removed it.

What do you think?

Comment (ishachopra0401) [reply destinations/advertising/google-adwords-enhanced-conversions.mdx:66]: oh.. got it! I will keep it under Event Settings then

Comment (ishachopra0401) [reply destinations/advertising/google-adwords-enhanced-conversions.mdx:66]: done

**Review by ItsSudip — APPROVED** [review]

**Pull request merged**